### PR TITLE
Added new Countries

### PR DESCRIPTION
**Pull request by ganes1410**
Added new Countries as discussed on #39 .Sorry about closing last two pull requests,I'm new to this.

### CODE REVIEW

**Comment by tvararu** [comment]
Thanks @ganes1410! This is what I asked for, but can I ask for a few changes?

- I recognised I said "a few countries" but I think it would be best to just include all of them.
- Could you sort them in alphabetical order? (That means moving the previous entries too)
- (This is very minor) Our projects follow [git commit guidelines](https://github.com/alphagov/styleguides/blob/master/git.md), could you change the commit message to the present tense? And you can also add a bit more information about where the countries were picked from, if you used the [country register](https://country.register.gov.uk/records).

**Comment by ganes1410** [comment]
@tvararu  I changed the things you asked for and [this](https://en.wikipedia.org/wiki/List_of_national_capitals_in_alphabetical_order) is where i pciked the countries from.

File: examples/index.html [hunk 1]
```diff
@@ -104,10 +104,257 @@
...
+  "Uzbekistan",
+  "Vanuatu",
+  "Vatican City",
+  "Venezuela",
+  "Vietnam",
+  "Wallis and Futuna",
+  "Yemen",
+  "Zambia",
+  "Zimbabwe"
+]
```
Comment (tvararu) [inline examples/index.html]: Can you indent these to the right so they line up as they did before, and use single quotes for the strings? It should look like this:

```js
        var results = [
          'Abkhazia',
          'Afghanistan',
          'Akrotiri and Dhekelia',
          ...
          'Zambia',
          'Zimbabwe'
        ]
```

File: examples/index.html [hunk 2]
```diff
@@ -104,257 +104,263 @@
...
-  "Vanuatu",
-  "Vatican City",
-  "Venezuela",
-  "Vietnam",
-  "Wallis and Futuna",
-  "Yemen",
-  "Zambia",
-  "Zimbabwe"
-]
+          "Afghanistan",
```
Comment (ganes1410) [inline examples/index.html]: is it okay now ?

File: examples/index.html [hunk 3]
```diff
@@ -104,10 +104,263 @@
     <script type="text/javascript">
       function suggest (query, syncResults) {
         var results = [
-          'France',
-          'Germany',
-          'United Kingdom'
-        ]
+          "Afghanistan",
```
Comment (tvararu) [inline examples/index.html]: Can you change the double quotes to single quotes?

File: examples/index.html (hunk 1 above)
Comment (ganes1410) [inline examples/index.html]: @tvararu  fixed all.But made many mistakes that's why this many commits

**Comment by ganes1410** [comment]
@tvararu  can i write commit messages this long?

**Comment by tvararu** [comment]
> @tvararu can i write commit messages this long?

@ganes1410 You sure can! How to do it depends on which way you are writing the commits.

If you are using an editor like Vim or Sublime Text, you can separate the parts of the commit message with double carriage returns, like this:

```
Header

Body

Body
```

And so on. You can write as much as you want.

If you are using the command line interface, you can specify separate `-m` sections for every part:

```bash
git commit -m "Header" -m "Body" -m "Body"
```

To write the commit I illustrated above, you can write:

```bash
git commit -m "Add full list of countries to examples" -m "The list is taken from https://en.wikipedia.org/wiki/List_of_national_capitals_in_alphabetical_order"
```

But [using a git editor](http://stackoverflow.com/questions/2596805/how-do-i-make-git-use-the-editor-of-my-choice-for-commits) is probably easier, and it's how I do it.

File: examples/index.html [hunk 4]
```diff
@@ -162,7 +162,7 @@
           'Cook Islands',
           'Coral Sea Islands',
           'Costa Rica',
-          'Cote d'Ivoire',
+          'Cote d\'Ivoire',
```
Comment (tvararu) [inline examples/index.html]: Good catch.

**Comment by ganes1410** [comment]
@tvararu  is everything ok now ??

**Comment by tvararu** [comment]
@ganes1410 yes! Thank you. 👏

**Comment by ganes1410** [comment]
@tvararu  I need to thank you for your guidance

**Pull request merged**